src/page/liked/liked.module.css

```diff
@@ -0,0 +1,103 @@
+.cardContainer {
+  width: 75vw;
+  margin-left: 2rem;
+  display: flex;
+  flex-wrap: wrap;
+}
+.videoCard {
+  width: 17rem;
+  height: 15rem;
+  border: 1px solid var(--COLOR-BACKGROUND-LIGHTER);
+  margin: 0.5rem 1rem;
+  cursor: pointer;
+  position: relative;
+}
+.image {
+  width: 100%;
+  height: 60%;
+}
+.image:hover {
+  opacity: 0.6;
+}
+.videoDetail {
+  height: 40%;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+.videoThumbnail {
+  display: inline-block;
+  margin: 0 0.8rem;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 50%;
+}
+.videoTitle {
+  text-align: start;
+  font-size: medium;
+}
+.views {
+  display: block;
+  font-size: 0.7rem;
+}
+.videoDetailWrapper {
+  display: flex;
+  justify-content: space-between;
+}
+.iconWrapper > div {
+  padding: 0.7rem 0.9rem;
+  margin-right: 0.4rem;
+  margin-top: 0.2rem;
+}
+.iconWrapper > div:hover {
+  background-color: var(--COLOR-BACKGROUND-LIGHTER);
+  border-radius: 50%;
+}
+.toggleMenu {
+  background-color: var(--COLOR-BACKGROUND-LIGHTER);
+  display: flex;
```

Comment:
Make flex a helper class so that you don't have to write every time.

Reply:
thank you for pointing that out  but I am using `module.css` only